assets/model_monitoring/components/model_monitor/model_monitor_azmon_metric_publisher/spec.yaml

```diff
@@ -31,6 +31,7 @@ inputs:
     optional: True
 
 conf:
+  spark.hadoop.aml.enable_cache : "true"
```

Comment:
what does this spark.hadoop.aml.enable_cache do?

Reply:
it makes sure that if its a first job with this configuration then it triggers a package installation and stores in zip file. if the job is submitted again with same package, package installation is skipped and the zip file is used to place packages in the node so they can be used.